src/Options.py

```diff
@@ -203,6 +203,9 @@ def addOptionToGroup(option_name: str, group: str):
 
 for category in category_table:
     for option_name in category_table[category].get("yaml_option", []):
+        for c in "><=:": # Range and Choice options must be defined using Options.json
+            if c in option_name:
+                continue
```

Comment:
Can you make this check for `option_name in manual_options`, and throw an exception if they forgot to define it?

Reply:
Fixed in e5f27a8f6489ee0fc4211cb6f0a2a4c122c60c87